faraday/aprs.py

```diff
@@ -317,11 +382,8 @@ def sendPositions(stations, socket):
 
                 logger.debug(positionString)
 
-                try:
-                    socket.sendall(positionString)
-
-                except IOError as e:
-                    logger.error(e)
+                status = sendAPRSPacket(socket, positionString)
+                return status
```

Comment:
`return sendAPRSPacket(socket, positionString)`

Here and below.

Reply:
All `return status` lines replaced with obviously better `return sendAPRSPacket()`. Thanks for observing this unnecessary addition of variables!